### PR TITLE
Fix UG

### PR DESCRIPTION
**Pull request by willowisp01**

### CODE REVIEW

**Comment by willowisp01** [comment]
I've completed all my changes to the UG. Please have a look!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -491,7 +491,7 @@ Displayed after command: `summary`
 |------------------|------------------------------------------------------------------------------------------------------------------------------|
 | **Add**          | `add id/STUDENT_ID n/NAME e/EMAIL g/GRADE [gp/GROUP]...` <br> e.g., `add id/A01234567X n/James Ho e/e0123456@u.nus.edu g/B+` |
 | **Delete**       | `delete STUDENT_ID`<br> e.g., `delete A0123456X`                                                                             |
-| **Edit**         | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [gp/GROUP]...` <br> e.g.,`edit A0123466C g/A+`                   |
+| **Edit**         | `edit STUDENT_ID [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [g/GRADE] [gp/GROUP]...` <br> e.g.,`edit A0123466C g/A+`                   |
```
Comment (Wongqingyee) [inline docs/UserGuide.md:494]: I think we dont have address and phone number anymore for the params.

Comment (Wongqingyee) [reply docs/UserGuide.md:494]: We can also edit the student_id.

Comment (Wongqingyee) [reply docs/UserGuide.md:494]: edit STUDENT_ID [id/STUDENT_ID] [n/NAME] [e/EMAIL] [g/GRADE] [gp/GROUP_NAME]...

Comment (willowisp01) [reply docs/UserGuide.md]: I think the issue should be fixed now

File: docs/UserGuide.md [hunk 2]
```diff
@@ -219,24 +219,24 @@ Format: `list`
 * Displays student's email that is a clickable link to open the user's mailbox with a new email to the student's email address.
 
 
-### 7.4 Editing a student : `edit`
+### 7.4 Editing a person : `edit`
```
Comment (Wongqingyee) [inline docs/UserGuide.md:222]: i think it would be good to change it to student

Comment (willowisp01) [reply docs/UserGuide.md:222]: Yes, it seems that i did the merge conflict incorrectly. Please give me a moment

**Comment by willowisp01** [comment]
Actually it seems that some of us are already working on the same bugs so this branch is quite confusing now. I'll close this and open a new branch instead.

**Pull request closed without merging**